microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md

```diff
@@ -0,0 +1,183 @@
+Image -->
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+
+<!-- ====================================================================== -->
+## The 3D View tool responds better in narrow layouts
+
+<!-- Title: 3D View: Now with more responsive design -->
+<!-- Subtitle: The 3D View toolbar now wraps around in a narrower window. -->
+
+As folks use the **3D View** tool more, especially with the addition of Composited Layers (formerly known as **Layers**), we have found opportunities to improve the UI based on your feedback.  We've made the layout of the 3D View toolbar more responsive so that your development isn't hindered even when DevTools is narrow.  Keep the feedback coming as you check out 3D View!
+
+![Responsive layout of toolbar in 3D View tool.](devtools-103-images/3d-view-responsive-layout.png)
+
+See also:
+* [Navigate webpage layers, z-index, and DOM using the 3D View tool](../../../3d-view/index.md)
+
+
+<!-- ====================================================================== -->
+## Fix: The Close button for DevTools is now accessible in high-contrast mode
+
+<!-- Title: Fix: In high contrast mode, the Close button for DevTools is now accessible -->
+<!-- Subtitle: In previous versions of Microsoft Edge, the Close button was not visible in high contrast mode but this issue has been fixed in Microsoft Edge 103. -->
+
+In previous versions of Microsoft Edge, the Close button for DevTools didn't render correctly in high-contrast mode.  In Microsoft Edge 103, this issue has been fixed.  The **Close** button for DevTools is displayed correctly in high-contrast mode.
+
+<!-- ![alt text.](devtools-103-images/x.png) -->
+
+<!-- Instructions for screenshot
+(how to configure the DevTools correctly, a website or CodePen that the writer can use, where to navigate in the DevTools UI for the screenshot, etc.)
+Open Edge Stable and navigate to edge://version and ensure you're on version 102 or at least <103
+From Windows Settings, navigate to Accessibility > Contrast themes and select the Night sky theme to apply
+Open DevTools in Edge Stable. Note that the Close button won't be visible (you can still select it though).
+Refer to the attachment for reference
+Repeat the same steps in Edge Canary or Dev, just ensure the version is 103+. Refer to the attachment for reference. -->
+
+<!-- Video recording of feature in action
+.mp4 attachments -->
+
+See also:
+* [Navigate DevTools with assistive technology](../../../accessibility/navigation.md)
+* [Windows High contrast mode](/fluent-ui/web-components/design-system/high-contrast)
+
+
+<!-- ====================================================================== -->
+## Welcome tool redesign
```

Comment:
@Reezaali - ready for title/subtitle/ draft text.  rendered:
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103?branch=pr-en-us-2007#welcome-tool-redesign